diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4af6f90b7..7557a2f2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,6 +32,8 @@ add_subdirectory(plugins)
 
 add_subdirectory(test/test-input)
 
-add_subdirectory(frontend)
+# add_subdirectory(frontend)
+
+add_subdirectory(cli-capture)
 
 message_configuration()
diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index c12f015c8..fdec55b3d 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -31,18 +31,18 @@ macro(check_obs_websocket)
 endmacro()
 
 # Add plugins in alphabetical order to retain order in IDE projects
-add_obs_plugin(
-    aja
-    PLATFORMS WINDOWS MACOS LINUX
-    WITH_MESSAGE
-)
-add_obs_plugin(coreaudio-encoder PLATFORMS WINDOWS MACOS)
-add_obs_plugin(
-    decklink
-    PLATFORMS WINDOWS MACOS LINUX
-    WITH_MESSAGE
-)
-add_obs_plugin(image-source)
+# add_obs_plugin(
+#     aja
+#     PLATFORMS WINDOWS MACOS LINUX
+#     WITH_MESSAGE
+# )
+# add_obs_plugin(coreaudio-encoder PLATFORMS WINDOWS MACOS)
+# add_obs_plugin(
+#     decklink
+#     PLATFORMS WINDOWS MACOS LINUX
+#     WITH_MESSAGE
+# )
+# add_obs_plugin(image-source)
 add_obs_plugin(linux-alsa PLATFORMS LINUX FREEBSD OPENBSD)
 add_obs_plugin(linux-capture PLATFORMS LINUX FREEBSD OPENBSD)
 add_obs_plugin(linux-jack PLATFORMS LINUX FREEBSD OPENBSD)
@@ -56,7 +56,7 @@ add_obs_plugin(mac-videotoolbox PLATFORMS MACOS)
 add_obs_plugin(mac-virtualcam PLATFORMS MACOS)
 add_obs_plugin(nv-filters PLATFORMS WINDOWS)
 
-check_obs_browser()
+# check_obs_browser()
 
 add_obs_plugin(obs-ffmpeg)
 add_obs_plugin(obs-filters)
@@ -68,23 +68,23 @@ add_obs_plugin(
     PLATFORMS WINDOWS LINUX
     ARCHITECTURES x64 x86_64
 )
-add_obs_plugin(obs-text PLATFORMS WINDOWS)
-add_obs_plugin(obs-transitions)
-add_obs_plugin(
-    obs-vst
-    PLATFORMS WINDOWS MACOS LINUX
-    WITH_MESSAGE
-)
+# add_obs_plugin(obs-text PLATFORMS WINDOWS)
+# add_obs_plugin(obs-transitions)
+# add_obs_plugin(
+#     obs-vst
+#     PLATFORMS WINDOWS MACOS LINUX
+#     WITH_MESSAGE
+# )
 add_obs_plugin(obs-webrtc)
 
-check_obs_websocket()
+# check_obs_websocket()
 
 add_obs_plugin(obs-x264)
 add_obs_plugin(oss-audio PLATFORMS FREEBSD OPENBSD)
 add_obs_plugin(rtmp-services)
 add_obs_plugin(sndio PLATFORMS LINUX FREEBSD OPENBSD)
-add_obs_plugin(text-freetype2)
-add_obs_plugin(vlc-video WITH_MESSAGE)
+# add_obs_plugin(text-freetype2)
+# add_obs_plugin(vlc-video WITH_MESSAGE)
 add_obs_plugin(win-capture PLATFORMS WINDOWS)
-add_obs_plugin(win-dshow PLATFORMS WINDOWS)
+# add_obs_plugin(win-dshow PLATFORMS WINDOWS)
 add_obs_plugin(win-wasapi PLATFORMS WINDOWS)
